content/managing-apps.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Installing, Updating, and Removing Applications
+description: >
+  Quick guide to installing, updating, and removing apps on your device.
+keywords:
+  - Apps
+  - Install
+  - Update
+  - Remove
+  - Delete
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: pop
+tableOfContents: true
+---
+
+## Installing & Uninstalling Applications via Terminal
+
+Launch the Terminal `SUPER` + `T` to Install applications using a command line interface. Terminal commands instruct package managers to retrieve application installation packages from repositories.
+
+### Flatpaks & .Deb Files
+
+**Flatpaks** will install a “containerized” version of the software. This means the software runs in its own sandbox, and the installation will include all dependencies and libraries required by the application. Flatpaks pull all dependencies from flathub.org, while installing with apt will pull from Ubuntu repos.
+
+Substitute `[packagename]` with the desired application (do not include brackets in the command).
+
+Apt install:
+
+```bash
+apt install [packagename]
+```
+
+Apt uninstall:
+
+```bash
+sudo apt remove [packagename]
+```
+
+Apt uninstall (alternative):
+
+```bash
+sudo apt purge [packagename]
+```
+
+Flatpak install:
+
+```bash
+flatpak install [packagename]
+```
+
+Flatpak uninstallL
+
+```bash
+flatpak uninstall [packagename]
+```
+
+## Updating POP!\_OS via Terminal
+
+1. Open a terminal by pressing `SUPER` + `T` and type the following commands:
+
+```bash
+sudo apt update
+```
+
+```bash
+sudo apt dist-upgrade
```

Comment:
`dist-upgrade` is a deprecated command. Please use `full-upgrade` in support documents (that's the current name for it that does the same thing).

If you're not going to include any info in between these two commands, then they should be in the same code block instead of separate code blocks.